frontend/apps/reader/modules/readertypeset.lua

```diff
@@ -497,12 +497,13 @@ function ReaderTypeset:onSetPageMargins(margins, refresh_callback)
             text = T(_([[
 Margins set to:
 
-  horizontal: %1 (%2px)
-  top: %3 (%4px)
-  bottom: %5 (%6px)
+  left: %1 (%2px)
+  right: %3 (%4px)
+  top: %5 (%6px)
+  bottom: %7 (%8px)
 
 Tap to dismiss.]]),
-            margins[1], left, margins[2], top, margins[4], bottom),
+            margins[1], left, margins[3], right, margins[2], top, margins[4], bottom),
```

Comment:
Is the table really inverted? (e.g., shouldn't this be `left, margins[2], right, margins[3]` ?)

Reply:
Yes:
https://github.com/koreader/koreader/blob/a44c178d7423575b535e521a863c63398a1b360b/frontend/apps/reader/modules/readertypeset.lua#L92-L93
https://github.com/koreader/koreader/blob/a44c178d7423575b535e521a863c63398a1b360b/frontend/apps/reader/modules/readertypeset.lua#L433-L435

Reply:
Or, right, it's L/T/R/B, I read the formatting args in the wrong order ;).